collect_app/src/main/java/org/odk/collect/android/adapters/DeleteFormsTabsAdapter.java

```diff
@@ -24,7 +27,8 @@ public Fragment createFragment(int position) {
             case 0:
                 return new SavedFormListFragment();
             case 1:
-                return new BlankFormListFragment();
+                String className = DeleteBlankFormFragment.class.getName();
+                return fragmentFactory.instantiate(Thread.currentThread().getContextClassLoader(), className);
```

Comment:
I can't find any approaches to using ViewPager2 with `FragmentFactory` that don't feel a little "icky". I'm guessing that's an inconsistency in the SDK for the moment. We can maybe create a helper or an implementation of `FragmentStateAdapter` that makes this nicer as part of reimplementing Delete Saved Form.